### PR TITLE
COMPOSE-350 Remove old Gradle tasks to launch iOS App

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Issue https://youtrack.jetbrains.com/issue/COMPOSE-350

Also removed kotlinx-serialization library because we don't need to parse json output of `xrun simctl` anymore

### CODE REVIEW

**Review by FelixHennerich — COMMENTED** [review]
code remove

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt [hunk 1]
```diff
@@ -5,25 +5,13 @@
 
 package org.jetbrains.compose.experimental.dsl
 
-import org.gradle.api.Action
 import org.gradle.api.model.ObjectFactory
-import org.jetbrains.compose.internal.requiredDslProperty
 import javax.inject.Inject
 
 @Suppress("unused")
 abstract class ExperimentalUiKitApplication @Inject constructor(
```
Comment (igordmn) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt:12]: Do we still need `ExperimentalUiKitApplication` and `ExperimentalUiKitExtension`?

It seems there are not used now.

Comment (dima-avdeev-jb) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt:12]: Yes, we can remove it too

Comment (dima-avdeev-jb) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/dsl/ExperimentalUiKitApplication.kt]: Done

**Review by igordmn — APPROVED** [review]

**Pull request merged**